README.md

```diff
@@ -1,8 +1,51 @@
-# embedded-berm-bot
+# NASA Lunabotics
 
-Branch for testing the Arduino code
+This is a competition that is hosted every year by NASA and other sponsors. The current obejctive of the competition is to build a bot that is able to traverse a simulated lunar enviroment (baring low gravity), at the end of this traversal it will need to then collect the lunar simulant *[BP-1](https://ares.jsc.nasa.gov/projects/simulants/bp-1.html)*, *[LHS-1](https://spaceresourcetech.com/products/lhs-1-lunar-highlands-simulant)*, etc. once collected this material is then transported to a deposition/berm building area. The rules, guidelines, and documentation can be found [here](https://www.nasa.gov/learning-resources/lunabotics-challenge/).
 
-## MoonDawgArduino
+## embedded-berm-bot
 
-This is the the code that will be run on the robot.
-Make sure to follow the README in the "Libraries" folder to properly run and compile the file in the Arduino IDE
+For all things Arduino and Arduino-esque on the SIU Lunabotics bot. 
+Due to the nature of how our bot is constructed we have seperated the logic in two distinct places. 
+<details>
+  <summary>Raspberry Pi/Master</summary>
+  
+  This will control the Arduino and tells it what to do along with communicating to the user. There is a more detailed description of how it works [here](https://github.com/SIU-Robotics/moondawg-ros). All that needs to be known for the Arduino side is that it will send data over Serial which is then parsed into usable bits for us to send to `commandProccessing` that will call specific functions and give it the data needed.
+  
+</details>
+
+### Arduino/Slave
+
+#### Overview
+  
+This is what processes the commands sent from the Raspberry Pi/Master and once processed they trigger certain commands that control the bot. There is no feedback currently implemented so it is all manually controlled by a user. 
```

Comment:
what is "Raspberry Pi/Master"?

Reply:
It's the Pi, how the Arduino is controlled. Or should I say how the Arduino knows what functions to use based on the commands sent from the Pi

Reply:
I see, you are referring to the raspberry pi as the master, I thought you were referencing some kind of branch on a repository named raspberry pi lol

Reply:
Oh lol

Reply:
Same as comment above

Reply:
Done